test/lib/user.test.js

```diff
@@ -242,6 +242,28 @@ describe('User Model', () => {
         });
     });
 
+    describe('removeSettings', () => {
+        beforeEach(() => {
+            user.settings = {
+                1: {
+                    showPRJobs: true
+                },
+                11: {
+                    showPRJobs: false
+                },
+                displayJobNameLength: 25
+            };
+        });
+
+        it('Remove user settings', () => {
+            datastore.update.resolves({});
+
+            return user.removeSettings(settings).then(data => {
```

Comment:
`user.removeSettings(settings)` => `user.removeSettings()`

Latter does not take parameters